x/dsmr/node.go

```diff
@@ -207,6 +207,19 @@ func (n *Node[T]) NewBlock(parent Block, timestamp int64) (Block, error) {
 	return blk, nil
 }
 
+func (*Node[T]) Execute(ctx context.Context, _ Block, block Block) error {
```

Comment:
Isn't this supposed to be verify?

Reply:
This is meant to be renamed to `Execute` corresponding to https://github.com/ava-labs/hypersdk/pull/1744

`Verify` comes from AvalancheGo's `snowman.Block` interface and includes additional checks. The intent of `Execute` is that we are simply executing the state transition and assume that all extra information is supplied directly to the function ie. parent block, state, etc.